org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TargetPlatformProject.xtend

```diff
@@ -86,26 +86,52 @@ class TargetPlatformProject extends ProjectDescriptor {
 						<repository location="https://download.eclipse.org/modeling/tmf/xtext/updates/releases/�config.xtextVersion�/"/>
 					�ENDIF�
 				</location>
-				<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-					<unit id="com.google.gson" version="2.8.7.v20210624-1215"/>
-					<unit id="com.google.inject" version="5.0.1.v20210324-2015"/>
-					<unit id="javax.inject" version="1.0.0.v20091030"/>
-					<unit id="org.antlr.runtime" version="3.2.0.v201101311130"/>
-					<unit id="org.junit" version="4.12.0.v201504281640"/>
+				�IF config.javaVersion.isAtLeast(JavaVersion.JAVA11)�
```

Comment:
am not sure if this is a good idea

Reply:
Or is it disallowed to have the same location twice?

Reply:
have no idea, we also would need 3-4 variables